week2/homework/src/remove.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+
+function remove(todoTask) {
+  if (todoTask !== undefined) {
+    fs.readFile('./data.txt', 'utf-8', (err, data) => {
+      if (err) throw err;
```

Comment:
Imagine trying to like post of a friend on Instagram and seeing the following error:
```
Error: Cannot write permissions of '\\Home-pc\e\MA MUSIQUE\DIVERS\data.txt.
Windows Error Code 1314: A required privilege is not held by the client. 
```

You were just trying to like a post what does this have to do with files and privileges. That is how the user of the todo app will feel. Here is an example of an uncaught error showing up on a dutch government website
![image](https://user-images.githubusercontent.com/3776580/63950663-b4d31a80-ca7c-11e9-96db-3a399fb9a648.png)
and here is one where the stacktrace is printed on an invoice
![image](https://user-images.githubusercontent.com/3776580/63950784-eb109a00-ca7c-11e9-8c8d-1e8230379cbb.png)

Conclusion: always wrap your errors.

Reply:
Thanks for the detailed explanation and examples @gajduk.  I made a small change on how I handle the errors. I don't know if this is enough or if I need more exhaustive error handling.